packages/ozone-helper/ozone-search-helper/src/ozone-search-helper.ts

```diff
@@ -47,6 +44,9 @@ export type BasicOzoneType = string | boolean | number
  * ```
  */
 export class SearchQuery {
+	private size = 10
```

Comment:
For me there's no sense to have those properties
If I understand correctly, those properties don't have any "impact" on the searchQuery.
Right ?

Reply:
Size is used in suggestion()
https://github.com/taktik/ozone-components/pull/107/files/8282eca3a906276e3f064f4cb6aaee1425a63441#diff-c26d387fbd9266f93e8e596de6de8288b6fc6fda1598e69b6fd973802162085aR277

Reply:
But for offset we don't need it indeed